--- a/app/Providers/AppServiceProvider.php
+++ b/app/Providers/AppServiceProvider.php
@@ -12,6 +12,7 @@
 use Illuminate\Support\Arr;
 use Illuminate\Support\Number;
 use Illuminate\Support\ServiceProvider;
+use Illuminate\Support\Facades\URL;
 use NumberFormatter;
 
 class AppServiceProvider extends ServiceProvider
@@ -35,6 +36,8 @@
      */
     public function boot(): void
     {
+        URL::forceScheme('https');
+
         FilamentColor::register([
             'primary' => Color::Stone,
             'gray' => Color::Zinc,
